__tests__/app.test.js

```diff
@@ -85,3 +83,61 @@ describe("GET review by id, api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("Update review by review_id, PATCH /api/reviews/:review_id", () => {
+  test(`Status 201 - Returns with the updated review, when provided with a review_id and a new Vote object`, () => {
+    const newVote = { inc_votes: 3 };
+
+    //votes has 5 originally, adds 3 from inc_votes
+    const expected = {
+      review_id: 2,
+      title: "Jenga",
+      designer: "Leslie Scott",
+      owner: "philippaclaire9",
+      review_img_url:
+        "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+      review_body: "Fiddly fun for all the family",
+      category: "dexterity",
+      created_at: "2021-01-18T10:01:41.251Z",
+      votes: 8,
+    };
+
+    return request(app)
+      .patch("/api/reviews/2")
+      .send(newVote)
+      .expect(201)
+      .then(({ body: { updatedReview } }) => {
+        expect(updatedReview).toEqual(expected);
+      });
+  });
+  test("404 - Valid number in path but doesn't match a review", () => {
+    const newVote = { inc_votes: 3 };
+    return request(app)
+      .patch("/api/reviews/999999")
+      .send(newVote)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Route not found");
+      });
+  });
+  test("400 - Something that is not a number is passed as the id", () => {
+    const newVote = { inc_votes: 3 };
+    return request(app)
+      .patch("/api/reviews/parrots")
+      .send(newVote)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("400 - Something that is not a number is passed as the inc_votes amount", () => {
+    const newVote = { inc_votes: "stop breaking my test" };
+    return request(app)
+      .patch("/api/reviews/2")
+      .send(newVote)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
```

Comment:
not considered if inc_votes is `undefined`. How would you like your server to respond?